src/template-strings/info.js

```diff
@@ -1,21 +1,21 @@
 const { wrapTemplates } = require('../utils/general')
 
 const infoMsg = {
-  createdApi: 'Created API \'{{owner}}/{{name}}\'',
+  ApiCreate: 'Created API \'{{owner}}/{{name}}\'',
```

Comment:
Not sure about about the usage of "PascalCase" here but I wanted some way to differentiate "one to one" command success / error messages, from other general messages